neomodel/match.py

```diff
@@ -7,13 +7,24 @@
 
 def rel_helper(**rel):
     if rel['direction'] == OUTGOING:
-        stmt = '-[{0}:{1}]->'
+        stmt = '-{0}->'
     elif rel['direction'] == INCOMING:
-        stmt = '<-[{0}:{1}]-'
+        stmt = '<-{0}-'
     else:
-        stmt = '-[{0}:{1}]-'
-    ident = rel['ident'] if 'ident' in rel else ''
-    stmt = stmt.format(ident, rel['relation_type'])
+        stmt = '-{0}-'
```

Comment:
I think there is no penalty, but there is a logical difference implied by a relationship direction. i.e. if the relationships were defined having a direction, the simple truth is that ```(x)--(y)``` and ```(x)-->(y)``` will have different results, so changing this would be breaking. it is possible however to do `rel.get('direction')` so it is not mandatory: defaulting to `(x)--(y)`

P.S. how about changing the func definition to `def rel_helper(lhs, rhs, ident=None, relation_type=None, direction=None)` so mandatory and optiona params are explicit?